grpc/src/main/java/io/stargate/grpc/payload/PayloadHandlers.java

```diff
@@ -0,0 +1,15 @@
+package io.stargate.grpc.payload;
+
+import io.stargate.grpc.payload.cql.ValuesHandler;
+import io.stargate.proto.QueryOuterClass.Payload;
+import io.stargate.proto.QueryOuterClass.Payload.Type;
+import java.util.EnumMap;
+
+public class PayloadHandlers {
+  public static final EnumMap<Payload.Type, PayloadHandler> HANDLERS =
```

Comment:
Maybe it's better to expose the interface here `Map<Payload.Type, PayloadHandler>` ? ps this is modifiable from anybody, maybe better init in the `static` and wrap with unmodifiable